package-overrides/npm/moment@2.0.0.json

```diff
@@ -1,3 +1,4 @@
 {
-  "jspmNodeConversion": false
+  "jspmNodeConversion": false,
+  "format": "umd"
```

Comment:
This isn't an option in jspm! Perhaps `format: 'cjs'` or `format: 'amd'` is what you're after?